lib/webhook.js

```diff
@@ -44,14 +44,23 @@ class AssetComputeWebhookEvents {
             event:event.payload || {}
         });
         let signature;
-        try{
-            console.log('generating HMAC signature:');
-            signature = this.hmacPrivateKey ? generateHMACSignature(eventBody, this.hmacPrivateKey) : undefined;
-        } catch(error) {
-            const message = `Error generating HMAC signature: ${error}`;
-            console.error(message);
-            throw Error(message);
-        } 
+        
+        if(this.hmacPrivateKey) {
+            const decodeHmacPrivateKey = Buffer.from(this.hmacPrivateKey, 'base64');
```

Comment:
https://nodejs.org/api/buffer.html#static-method-bufferfromstring-encoding A TypeError will be thrown if string is not a string or another type appropriate for Buffer.from() variants. @pheenomenon is there a possibility of this failing and we need to handle exception?